src/containers/Dev/UserSelection.tsx

```diff
@@ -40,9 +40,9 @@ const UserSelection: React.FC = () => {
     } else localStorage.setItem(config.localStorageJWTKey, loginResult.JWT)
 
     // Organisation login (auto-select first in list)
-    const { JWT, user, templatePermissions, orgList, isAdmin } = loginResult
+    const { JWT, user, templatePermissions, permissionNames, orgList, isAdmin } = loginResult
```

Comment:
We need a simple list of "permissionNames" in order to determine if the current user has "management" access. Plus, it'll probably useful in other places too, I'd imagine.

Reply:
I could have just returned `isManager` from back-end like `isAdmin`, but thought a list of currently active permission names is probably useful for other things. (e.g. It'll automatically be available to evaluator expressions as it's in the "currentUser" state, so we can make form elements conditional on having a given permission, say)